test/integration/user_creates_an_order_and_cart_empties_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.